src/rdkafka.h

```diff
@@ -1727,7 +1727,63 @@ void rd_kafka_conf_set_stats_cb(rd_kafka_conf_t *conf,
 						  size_t json_len,
 						  void *opaque));
 
+/**
+ * @brief Set SASL/OAUTHBEARER token refresh callback in provided conf object.
+ *
+ * The SASL/OAUTHBEARER token refresh callback is triggered via rd_kafka_poll()
+ * whenever OAUTHBEARER is the SASL mechanism and a token needs to be retrieved,
+ * typically based on the configuration defined in \c sasl.oauthbearer.config.
+ * Function arguments:
+ *   - \p rk - Kafka handle
+ *   - \p opaque - Application-provided opaque.
```

Comment:
I think the name `opaque` is a bit... opaque. As a user, I wouldn't easily understand what I'd be passing here.

Reply:
This is a `librdkafka`-ism rather than something specific to this PR; it's how all the callbacks work.  I'll add the text `set via rd_kafka_conf_set_opaque()` for additional clarity.